### ISSUE TITLE
Apparent resistivity for homogeneous halfspace

### ISSUE THREAD

**Issue by jiajiasun** [geoscixyz/em#534]
I calculated the electric potential and the apparent resistivity for a homogeneous halfspace of 500 ohm*m using the equation and model setup shown in the following slide (taken from the DISC course). But, my results do not match what is shown on the slide.

![apparentresistivity](https://user-images.githubusercontent.com/379368/44807223-9c40d380-ab8e-11e8-805a-6fc6cc01910d.PNG)

Here is how I calculated the potential at a location 10 meters away from the injection point which is located at 0 m, according to the above picture. V = rho * I/(2 * pi * r) = 500 * 1/(2 * pi * 10) = 7.9577 Volts (Here I assume the inject current is 1 A). But in the above picture, 6.9 Volts is shown. 

I also calculated the apparent resistivity: rho = 2 * pi * r * V / I = 2 * pi * 10 * 6.9/1 = 433.5398 Ohm * m. Shouldn't it be 500 ohm * m (because we are talking about homogeneous halfspace)?

Would you please double check what is wrong here? I would like my students to do similar calculations tomorrow in class. Thanks!

**Comment by jiajiasun** [comment]
This is a

**Comment by lheagy** [comment]
Hi @jiajiasun, this is a subtle point. I believe we created that with an earlier version of the 2.5D DC layered earth app (DC_Layer_Cylinder_2_5D.ipynb)

![image](https://user-images.githubusercontent.com/6361812/44820427-bafe9480-aba5-11e8-866b-d9ced075e14a.png)

In this case, it looks like we are not necessarily satisfying the boundary conditions (the mesh was trimmed to speed up the computation), so a [correction factor](https://github.com/geoscixyz/em_examples/blob/master/em_examples/DCWidgetResLayer2_5D.py#L394) is used under the hood so that the halfspace apparent resistivity is correct (cc @thast). 

With the image @thast showed above, it looks like we are doing a better job of mesh design in the plate app, so you might want to check out the 2.5D plate app if you would like the students to calculate it by hand.

**Comment by thast** [comment]
Seems also the panels are no longer aligning in some DC apps :/ I created an issue 
https://github.com/geoscixyz/em-apps/issues/54

**Comment by jiajiasun** [comment]
Hi, @thast @lheagy Thank you very much for your response and explanation! It now makes more sense! Following your explanation, I ran the two apps: DC_Plate2_5D and DC_Layer_Cylinder_2_5D, and was able to reproduce what @thast showed (i.e., 7.6e+00 V). Thanks again!

I also ran another app in DC_Cylinder_2D.ipynb with the same set-up, i.e., pole-pole survey with electrodes located at -0.25 and 10.25 m for a homogeneous halfspace of 500 ohm*m. And I got something interesting.

![dc_cylinder_2d](https://user-images.githubusercontent.com/379368/44821626-716d7400-abbc-11e8-9db7-a521b2949345.png)

The potential at the M electrode is 360 Volts. 

Is that because this is a 2D simulation (as compared to 2.5D or 3D)? How to make sense of 360 V?


Once again, thank you very much! All the materials and apps you have spent so much time on are super helpful. My students are benefiting from all your efforts. I posted the slides that I have used so far in class in my Azure library (https://notebooks.azure.com/jsunatuh/libraries/uhelectromagnetics). I will keep posting more slides and all the lab exericses as I move on.
